PWGHF/TableProducer/treeCreatorToXiPi.cxx

```diff
@@ -75,21 +68,17 @@ DECLARE_SOA_COLUMN(PxNegV0Dau, pxNegV0Dau, float);
 DECLARE_SOA_COLUMN(PyNegV0Dau, pyNegV0Dau, float);
 DECLARE_SOA_COLUMN(PzNegV0Dau, pzNegV0Dau, float);
 DECLARE_SOA_COLUMN(ImpactParCascXY, impactParCascXY, float);
-DECLARE_SOA_COLUMN(ImpactParPrimaryPiXY, impactParPrimaryPiXY, float);
+DECLARE_SOA_COLUMN(DcaXYToPvPiFromCharm, dcaXYToPvPiFromCharm, float);
```

Comment:
what about keeping the older name, since for the cascade you still talk about `ImpactPar` rather than `dcaXY`?